src/utils/SVGIcons.ts

```diff
@@ -0,0 +1,24 @@
+import { $$, Dom } from './Dom';
+
+declare var require: (svgPath: string) => string;
+
+export class SVGIcons {
+  public static search = require('svg/search');
+  public static more = require('svg/more');
+  public static facetLoading = require('svg/facet-loading');
+  public static checkboxMoreValues = require('svg/checkbox-hook-exclusion-more');
+  public static arrowUp = require('svg/arrow-up');
+  public static arrowDown = require('svg/arrow-down');
+
+  private static parser = new DOMParser();
+
+  public static addClassToSVGInContainer(svgContainer: HTMLElement, classToAdd: string) {
```

Comment:
Also, I understand this may look too specific as a method, but we will need a container everytime we use a svg icon. I think this is one of the very few operations we will need to do with svgs.